### PR TITLE
improve styling, add error message, sync UI after user is added

### PR DESCRIPTION
**Pull request by brodeynewman**

### CODE REVIEW

**Comment by nathanjohnson320** [comment]
looks good, could also write some tests if you want to https://github.com/nathanjohnson320/team-manager/blob/master/test/team_manager_web/live/user/live_user_signup_test.exs

File: lib/team_manager_web/live/tenant/manage.ex [hunk 1]
```diff
@@ -39,12 +41,14 @@ defmodule TeamManagerWeb.Live.Tenant.Manage do
...
-    else
-      _e ->
-        {:noreply, assign(socket, new_user: @new_user, show_user_modal: false)}
+  def handle_event("add_user", _, %{assigns: %{new_user: user, tenant: tenant, users: users}} = socket) do
+    case Accounts.add_user_to_tenant(user.email, tenant) do
+      {:ok, tu} ->
+        {:ok, added_user} = Accounts.get_user(tu.user_id)
+        {:noreply, assign(socket, new_user: tu, show_user_modal: false, users: [added_user | users] )}
+
+      _ ->
```
Comment (nathanjohnson320) [inline lib/team_manager_web/live/tenant/manage.ex]: we could match on the error cases to show specific messages like "user doesn't exist" and then use the @generic_error in the not matching case

Comment (brodeynewman) [reply lib/team_manager_web/live/tenant/manage.ex]: Done

File: lib/team_manager_web/templates/tenant/manage.html.leex [hunk 2]
```diff
@@ -72,6 +72,9 @@
             <%= email_input(:user, :email, [class: "input", placeholder: "Email", autocomplete: "off"]) %>
           </div>
         </div>
+        <p class="help is-danger">
```
Comment (nathanjohnson320) [inline lib/team_manager_web/templates/tenant/manage.html.leex]: would it make sense to wrap this in an if statement?

**Pull request merged**